### PR TITLE
fix: Split SimOptions Settings page into SimOptions and Realism page

### PR DESCRIPTION
**Pull request by frankkopp**
## Screenshots (if necessary)
Split SimOptions Settings page into SimOptions and Realism page

## References
As in current Development

Discord username (if different from GitHub): Cdr_Maverick#6475

## Testing instructions
-

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: src/instruments/src/EFB/Settings/Pages/RealismPage.tsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
+import React, { } from 'react';
```
Comment (ErickSharp) [inline src/instruments/src/EFB/Settings/Pages/RealismPage.tsx:1]: Remove the dangling brackets
```suggestion
import React from 'react';
```

File: src/instruments/src/EFB/Settings/Pages/RealismPage.tsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+        { name: 'Real', setting: 'REAL' },
+    ];
+
+    const steeringSeparationButtons: (ButtonType & SimVarButton)[] = [
+        { name: 'Disabled', setting: '0', simVarValue: 0 },
+        { name: 'Enabled', setting: '1', simVarValue: 1 },
+    ];
+
+    return (
+        <div>
```
Comment (ErickSharp) [inline src/instruments/src/EFB/Settings/Pages/RealismPage.tsx:48]: Remove these enclosing div tags

File: src/instruments/src/EFB/Settings/Pages/RealismPage.tsx [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+            <SettingsPage name="Sim Options">
```
Comment (ErickSharp) [inline src/instruments/src/EFB/Settings/Pages/RealismPage.tsx:49]:
```suggestion
            <SettingsPage name="Realism">
```

**Review by ErickSharp — CHANGES_REQUESTED** [review]

File: src/instruments/src/EFB/Settings/Pages/RealismPage.tsx [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+        { name: 'Real', setting: 'REAL' },
+    ];
+
+    const steeringSeparationButtons: (ButtonType & SimVarButton)[] = [
+        { name: 'Disabled', setting: '0', simVarValue: 0 },
+        { name: 'Enabled', setting: '1', simVarValue: 1 },
+    ];
+
+    return (
+        <>
```
Comment (ErickSharp) [inline src/instruments/src/EFB/Settings/Pages/RealismPage.tsx:48]: Remove these React fragments

**Review by ErickSharp — CHANGES_REQUESTED** [review]

File: src/instruments/src/EFB/Settings/Pages/RealismPage.tsx (hunk 4 above)
Comment (frankkopp) [reply src/instruments/src/EFB/Settings/Pages/RealismPage.tsx:48]: But they were there before and they are need, right? I move the once for the realism settings here and removed them in SimOptions.

Comment (ErickSharp) [reply src/instruments/src/EFB/Settings/Pages/RealismPage.tsx:48]: No, they were needed in the Sim Options page because of the conditional rendering.

**Review by ErickSharp — APPROVED** [review]

**Pull request merged**